src/ui/FacetRange/FacetRange.ts

```diff
@@ -97,6 +97,7 @@ export class FacetRange extends Facet implements IComponentBindings {
   };
 
   public options: IFacetRangeOptions;
+  public isFieldValueCompatible = false;
```

Comment:
Should other facets like the hierarchical, category, timespan etc. also define this property?

Reply:
No since they aren't compatible with fieldValue. It will be false/undefined for those.